splunk_eventgen/lib/plugins/output/counter.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import datetime
+import pprint
+import sys
+
+from outputplugin import OutputPlugin
+
+
+class CounterOutputPlugin(OutputPlugin):
```

Comment:
How do we use this good stuff?

Reply:
if we want to use this dummy counter plugin, we can set outputMode=counter in eventgen conf stanza. 
I did not update the readme doc because I think this is only used for debugging purpose. Do we have any place to hold the docs for developers? I can add some content to that place.

Reply:
What's wrong with exposing this option to customers?